datacube_alchemist/_utils.py

```diff
@@ -33,19 +29,24 @@ def compute(self, data) -> Dataset:
 
 
 def _get_logger(name: str):
-    logger = structlog.get_logger(name)
-    logging.basicConfig(
-        format="%(asctime)s %(name)s %(levelname)s %(message)s",
-        stream=sys.stdout,
-        level=logging.INFO,
-        datefmt="%y-%m-%d %H:%M:%S",
-    )
+    processors = [
+        structlog.stdlib.add_log_level,
+        structlog.processors.TimeStamper(fmt="%Y-%m-%d %H:%M:%S"),
+        structlog.processors.StackInfoRenderer(),
+        structlog.processors.format_exc_info,
+        structlog.dev.ConsoleRenderer(),
+    ]
+
     structlog.configure(
-        processors=[structlog.processors.KeyValueRenderer(key_order=["event"])],
-        context_class=structlog.threadlocal.wrap_dict(dict),
-        logger_factory=structlog.stdlib.LoggerFactory(),
+        processors=processors,
+        context_class=dict,
+        cache_logger_on_first_use=True,
+        logger_factory=structlog.PrintLoggerFactory(),
     )
 
+    logger = structlog.get_logger()
```

Comment:
I think it's a better pattern to run this line at the top of each code module directly.

Reply:
Sure thing